rxandroidble/src/main/java/com/polidea/rxandroidble/internal/connection/ConnectorImpl.java

```diff
@@ -55,9 +56,11 @@ public ConnectorImpl(
                     return Observable.error(new BleDisconnectedException(bluetoothDevice.getAddress()));
                 }
 
-                final ConnectionComponent connectionComponent = connectionComponentBuilder.build();
+                final ConnectionComponent connectionComponent = connectionComponentBuilder
+                        .connectionModule(new ConnectionModule(options.suppressOperationCheck))
+                        .build();
                 RxBleRadioOperationConnect operationConnect = connectionComponent.connectOperationBuilder()
```

Comment:
It should be now possible to get rid of the `RxBleRadioOperationConnect.Builder` as all dependencies could be passed through `ConnectionComponent.Builder`.